src/lang/en-US.json

```diff
@@ -64,7 +64,6 @@
   "api_month": "Month",
   "api_header_key": "Your Key",
   "api_header_table": "Get started for free. Keep going at a ridiculously low price.",
-  "api_header_details": "Details",
```

Comment:
why is this line removed?

Reply:
that line is a duplicate. line 49 and line 67

Reply:
thanks, maybe we should add a build step to check for duplicates